openapi/v2/support-notifications.yaml

```diff
@@ -35,10 +35,7 @@ components:
           description: "Identifies the sender of a notification, usually an email address."
           type: string
         severity:
-          description: "Indicates the level of severity for the notification. Current accepted values include: NORMAL, CRITICAL"
-          type: string
-        slug:
-          description: "Effectively a name or key that labels the notification."
+          description: "Indicates the level of severity for the notification. Current accepted values include: MINOR, NORMAL, CRITICAL"
```

Comment:
Shouldn't `AddNotificationRequest` just use `Notification` similar to all the other Request DTOs:
Example:
```
type AddDeviceServiceRequest struct {
	common.BaseRequest `json:",inline"`
	Service            dtos.DeviceService `json:"service"`
}
```

Reply:
Modified.